README.md

```diff
@@ -85,6 +85,7 @@ The distributed sessions module can be configured through the environment variab
 | session.DatastoreStore.namespace    |  Namespace to use in the Datastore.                         |  tomcat-gcp-persistent-session |
 | session.DatastoreStore.sessionKind  |  Name of the entity used to store sessions in the Datastore. |  TomcatGCloudSession |
 | session.DatastoreStore.sessionMaxInactiveTime |  Defines the maximum time (in seconds) a session can be inactive before being deleted by the expiration process. | 3600 |
+| session.DatastoreValve.uriExcludePattern | [Pattern](https://docs.oracle.com/javase/8/docs/api/java/util/regex/Pattern.html) specifying which Uri to ignore when persisting sessions. | null |
```

Comment:
+1 to changing `Uri` -> `URI`. 

Also, an example of configuring the `TOMCAT_PROPERTIES` env variable in app.yaml would be really helpful here. For example, I'd wonder how to configure multiple properties. Should they be comma-separated? Semicolon-separated? etc.